app/src/main/java/com/tari/android/wallet/ui/component/networkStateIndicator/ConnectionIndicatorViewModel.kt

```diff
@@ -45,13 +45,9 @@ internal class ConnectionIndicatorViewModel : CommonViewModel() {
                     TorProxyState.Initializing -> ConnectionIndicatorState.Disconnected(R.string.connection_status_error_unknown_network_connection_status)
                     TorProxyState.NotReady -> ConnectionIndicatorState.Disconnected(R.string.connection_status_error_connecting_with_tor)
                     is TorProxyState.Running -> {
-                        when (val validationResult = _baseNodeState.value) {
-                            is BaseNodeState.SyncCompleted -> {
-                                when (validationResult.isSuccess) {
-                                    true -> ConnectionIndicatorState.Connected(R.string.connection_status_ok)
-                                    false -> ConnectionIndicatorState.ConnectedWithIssues(R.string.connection_status_warning_sync_failed)
-                                }
-                            }
+                        when (_baseNodeState.value) {
+                            is BaseNodeState.Online -> ConnectionIndicatorState.Connected(R.string.connection_status_ok)
+                            is BaseNodeState.Offline -> ConnectionIndicatorState.ConnectedWithIssues(R.string.connection_status_warning_sync_failed)
```

Comment:
Isn't `Offline` equals to disconnected status?

Reply:
Good point, changed